components/dataList.js

```diff
@@ -469,7 +469,27 @@ export default [
         }
       ]
     }
+  },
+  {
+    id: 13,
+    searchKey: "Dengue",
```

Comment:
change the search to "dengue".
It is used as a part of the URL in other components.